### DISCUSSION TITLE
bundler: command not found: fastlane

### DISCUSSION THREAD

**Discussion by famasf1** [fastlane/fastlane#21970] [General]
I have issue installing fastlane using bundler on Windows 11. When i tried to run ```bundle exec fastlane -v``` or ```bundle exec fastlane init``` the error return

```
bundler: command not found: fastlane
Install missing gem executables with `bundle install`
```

I already run ```gem install bundler``` and ```bundle install```. I also already have Gemfile in my project's root dir with the following content.

```
source "https://rubygems.org"
gem 'fastlane', '~> 2.220'
```

But it's still not working. Is it env path-related? i did tried ```gem install fastlane``` then ```fastlane -v``` but i got the following error

```
fastlane: The term 'fastlane' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

Obviously without being able to run fastlane. I can't provided ```fastlane env``` output. So i have no idea what to do next. While waiting for any assistance i'll try using WSL to run fastlane in the meantime.

EDIT : WSL worked perfectly. But then i had to setup everything again in Linux environment which is kinda painful. So if anyone can guide me on fixing windows i'd be appreciated.

EDIT2 : this is log from ```bundle info fastlane```

```
  * fastlane (2.220.0)
        Summary: The easiest way to build and release mobile apps.
        Homepage: https://fastlane.tools
        Documentation: https://docs.fastlane.tools/
        Source Code: https://github.com/fastlane/fastlane
        Changelog: https://github.com/fastlane/fastlane/releases
        Bug Tracker: https://github.com/fastlane/fastlane/issues
        Path: C:/Users/User3/.local/share/gem/ruby/3.2.0/gems/fastlane-2.220.0
```

**Comment by lucgrabowski** [comment]
Hi, I converted it to a discussion so that the community could help you. This is not a fastlane tool issue.  Please use issues for bug reports and regressions.
Cheers.

**Comment by aminkhan1** [comment]
I have exactly the same problem in Windows
